usdk: Add starfive VisionFive board support

shalasaka opened the pull request:
Now usdk support fit image build for three starfive board:
starlight, starlight-a1 and visionfive.

You can choose your target board with assigning HWBOARD,
eg. "make HWBOARD=visionfive", or just default starlight-a1.
You can also specify the target to build, e.g. "make visionfive"

Signed-off-by: Som.Qin <som.qin@starfivetech.com>

Commit:
usdk: Add starfive VisionFive board support

Now usdk support fit image build for three starfive board:
starlight, starlight-a1 and visionfive.

You can choose your target board with assigning HWBOARD,
eg. "make HWBOARD=visionfive", or just default starlight-a1.
You can also specify the target to build, e.g. "make visionfive"

Signed-off-by: Som.Qin <som.qin@starfivetech.com>

MichaelZhuxx commented on Makefile:
```diff
@@ -2,6 +2,7 @@ ISA ?= rv64imafdc
 ABI ?= lp64d
 TARGET_BOARD := U74
 BOARD_FLAGS	:=
+HWBOARD ?= starlight-a1
```
VisionFive is recommended as the default since it is about to be mass-produced and sold.

Commit:
usdk: Update VisionFive the default target board of usdk build

Make VisionFive the default target board since it is about to be mass-produced.
Then update README.md with the build instructions.

Signed-off-by: Som.Qin <som.qin@starfivetech.com>

shalasaka commented:
Make VisionFive the default target board since it is about to be mass-produced.
Then update README.md with the build instructions.

Signed-off-by: Som.Qin som.qin@starfivetech.com

andyhu-stf commented:
Thanks shalasaka. It look ok. I will merge and close the pr.
